src/container/styles.scss

```diff
@@ -218,6 +218,7 @@
       &::before {
         content: '';
         border-bottom: none;
+        pointer-events: none;
```

Comment:
This would only fix it in the case when there is a high contrast header. Would this not also need to be applied above to the pseudo element block on line 192?

Reply:
Yep, updating!